Automattic/wp-calypso
# Stats: Updating Subscribers graph

Related to #75932

## Proposed Changes

* Updating graph gradient
* Using `spline` line interpolation for smooth edges

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug`
* Add the query string `?flags=stats/subscribers-section` to enable the subscriber counts page
* Verify that the subscribers chart loads and transitions between periods

| Before | After |
| --- | --- |
| ![SCR-20230504-pgqv](https://user-images.githubusercontent.com/112354940/236118661-a13d69c0-a432-4236-a87f-836e7738bdba.png) | ![SCR-20230504-pgvt](https://user-images.githubusercontent.com/112354940/236118673-0d5bd23f-7506-4862-87ac-77cb73aeb440.png) |



## Pre-merge Checklist
